hw/top_earlgrey/syn/chip_earlgrey_asic.ot.sdc

```diff
@@ -0,0 +1,1570 @@
+set_false_path -from [get_clocks SPI_TPM_OUT_CLK] \
+    -to [get_ports ${SPI_DEV_DATA_PORTS}] \
+    -through [get_pins -leaf -filter "@pin_direction == out" -of_objects \
+               [get_nets -segments -of_objects \
+                 [get_pins top_earlgrey/u_spi_device/u_p2s/s_o*]]]
+set_false_path -from [get_clocks SPI_TPM_OUT_CLK] \
+    -to [get_ports ${SPI_DEV_DATA_PORTS}] \
+    -through [get_pins -leaf -filter "@pin_direction == out" -of_objects \
+               [get_nets -segments -of_objects \
+                 [get_pins top_earlgrey/u_spi_device/u_p2s/s_en_o*]]]
+set_false_path -from [get_clocks SPI_TPM_OUT_CLK] \
+    -to [get_ports ${SPI_DEV_DATA_PORTS}] \
+    -through [get_pins -leaf -filter "@pin_direction == out" -of_objects \
+               [get_nets -segments -of_objects \
+                 [get_pins top_earlgrey/u_spi_device/u_read_pipe_stg1/q_o*]]]
+set_false_path -from [get_clocks SPI_TPM_OUT_CLK] \
+    -to [get_ports ${SPI_DEV_DATA_PORTS}] \
+    -through [get_pins -leaf -filter "@pin_direction == out" -of_objects \
+               [get_nets -segments -of_objects \
+                 [get_pins top_earlgrey/u_spi_device/u_read_en_pipe_stg1/q_o*]]]
+set_false_path -from [get_clocks SPI_TPM_OUT_CLK] \
+    -to [get_ports ${SPI_DEV_DATA_PORTS}] \
+    -through [get_pins -leaf -filter "@pin_direction == out" -of_objects \
+               [get_nets -segments -of_objects \
+                 [get_pins top_earlgrey/u_spi_device/u_read_pipe_stg2/q_o*]]]
+set_false_path -from [get_clocks SPI_TPM_OUT_CLK] \
+    -to [get_ports ${SPI_DEV_DATA_PORTS}] \
+    -through [get_pins -leaf -filter "@pin_direction == out" -of_objects \
+               [get_nets -segments -of_objects \
+                 [get_pins top_earlgrey/u_spi_device/u_read_en_pipe_stg2/q_o*]]]
+
+##
+# Remove hold analysis from the following paths to ports. Even though the pins
+# can change before the prior data was latched upstream, their effect is held
+# back by other logic on SPI_TPM_OUT_CLK.
+# Note: The final output logic equation must not permit glitches in the presence
+# of changes on the listed pins. Otherwise, any hold time failures could be
+# real.
+##
+
+# This path is from locality logic that is on the *_IN_CLK domain and selects
+# between fixed values or the return-by-hw register value. The flopped bits
+# settle in the middle of the command/address phase, many cycles before the
+# data phase.
+set_false_path -hold -from [get_clocks SPI_TPM_IN_CLK] \
+    -to [get_ports ${SPI_DEV_DATA_PORTS}] \
+    -through [get_pins -leaf -filter "@pin_direction == out" -of_objects \
+               [get_nets -segments -of_objects \
+                 [get_pins top_earlgrey/u_spi_device/u_spi_tpm/miso_o]]]
+
+# Remove scan paths for timing analysis
+set_clock_sense -stop_propagation -clock SPI_TPM_CLK [get_pins -leaf -filter "@pin_direction == in" -of_objects [get_nets -segments -of_objects [get_pins u_ast/u_scan_clk/in_i*]]]
+
+# Should be approved
```

Comment:
Has this been approved?

Reply:
This comment is still in the updated version. And I'll leave an additional comment there.